sonar-scanner-engine/src/main/java/org/sonar/batch/sensor/SensorWrapper.java

```diff
@@ -59,6 +59,10 @@ public void analyse(Project module, org.sonar.api.batch.SensorContext context) {
 
   @Override
   public String toString() {
-    return descriptor.name() + (LOG.isDebugEnabled() ? " (wrapped)" : "");
+    if (descriptor.name() != null) {
+      return descriptor.name() + (LOG.isDebugEnabled() ? " (wrapped)" : "");
```

Comment:
Can you please remove this "wrapped" addition? Not sure this is useful now that new Sensor API is out of beta.